contracts/lido/LidoInsuranceTest.sol

```diff
@@ -0,0 +1,36 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.7.0;
+
+import "../AnteTest.sol";
+
+contract AnteLidoInsuranceTest is AnteTest("Make sure at least 0.5% of Lido stake is insured!") {
```

Comment:
You can use some other Ante Tests as a guide, but let's add some Solidity Natspec to the test: https://docs.soliditylang.org/en/develop/natspec-format.html